ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -75,6 +97,7 @@ const Footer = () => {
         data-testid="confirm-footer-confirm-button"
         onClick={onSubmit}
         size={ButtonSize.Lg}
+        danger={isDangerButton}
         disabled={
           ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
           mmiSubmitDisabled ||
```

Comment:
Hey @digiwand : I think will be useful to extract this logic to set danger button in a hook as footer component is growing a lot and also we plan to get rid of it once the new alert system is integrated.

Reply:
sounds good to me! thanks and updated [8fd55c6](https://github.com/MetaMask/metamask-extension/pull/24402/commits/8fd55c6b7f2c64acaa0974b141eaec8eb110583b)